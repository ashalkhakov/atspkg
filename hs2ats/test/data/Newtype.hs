module Newtype where

newtype IntType = IntType Int
