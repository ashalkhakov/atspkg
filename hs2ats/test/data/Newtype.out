vtypedef int_type = int
